src/EFCore.PG/Storage/Internal/Mapping/NpgsqlListTypeMapping.cs

```diff
@@ -79,21 +85,22 @@ protected override string GenerateNonNullSqlLiteral(object value)
                 if (i > 0)
                     sb.Append(',');
 
-                sb.Append(ElementMapping.GenerateSqlLiteral(list[i]));
+                sb.Append(_elementMapping.GenerateSqlLiteral(list[i]));
             }
 
             sb.Append("]::");
-            sb.Append(ElementMapping.StoreType);
+            sb.Append(_elementMapping.StoreType);
             sb.Append("[]");
             return sb.ToString();
         }
 
         #region Value Comparison
 
-        // Note that the value comparison code is largely duplicated from NpgsqlAraryTypeMapping.
-        // However, a limitation in EF Core prevents us from merging the code together, see
-        // https://github.com/aspnet/EntityFrameworkCore/issues/11077
-
+        /// <remarks>
+        /// Note that the value comparison code is largely duplicated from NpgsqlArrayTypeMapping.
+        /// However, a limitation in EF Core prevents us from merging the code together, see
+        /// https://github.com/aspnet/EntityFrameworkCore/issues/11077
+        /// </remarks>
```

Comment:
As above, I think it's for the editor only.

Reply:
Agree

Reply:
Reverted.